src/NodeCompiler/Exception/UnableToCompileNode.php

```diff
@@ -15,6 +15,12 @@
 
 class UnableToCompileNode extends LogicException
 {
+    /**
+     * @psalm-readonly
+     * @var string
```

Comment:
maybe `string|null`? an empty string seems misleading, and I know it was done mostly because of static analysis 😅